### ISSUE TITLE
Cannot make any [Function] signs. Appear blank.

### ISSUE THREAD

**Issue by Prxmium** [EssentialsX/Essentials#687]
Whenever trying to create [Buy] and [Sell] signs for my spawn, after entering info and clicking done, nothing shows up and they are blank. Also tried with Disposal signs and the same thing happened.

EssentialsX Config: http://pastebin.com/eMpUguPS

**Comment by SupaHam** [comment]
When you modify your config you must make sure to either reload the configuration file in game or restart the server.
You can reload essentials by typing `/ess reload`.

**Comment by Prxmium** [comment]
I am very aware of this. What does that have to do with my signs not working?

**Comment by SupaHam** [comment]
The signs work fine on EssentialsX as tested multiple times. If they do not work for you and you modified your config without reloading, that is your issue. Based on your config signs would be working just fine.

**Comment by Prxmium** [comment]
They _should_ be working just fine. I have restarted my server multiple times trying to fix this, so it cannot be for that reason.

**Comment by SupaHam** [comment]
Pastebin your startup log.

**Comment by Prxmium** [comment]
Startup log: http://pastebin.com/LgVeB0HN

**Comment by SupaHam** [comment]
Are you sure you have permissions to create signs?
Op yourself by typing `op Coreo` in the console.

**Comment by Prxmium** [comment]
I am opped and placed in a group with the \* node.

**Comment by SupaHam** [comment]
Please do the following:
1. type `/ess debug`
2. create [buy] sign
3. pastebin console log.

**Comment by Prxmium** [comment]
Log: http://pastebin.com/w2UKYG8E

**Comment by SupaHam** [comment]
Based on that, the sign is actually being created. Try right click the sign.

**Comment by Prxmium** [comment]
Still nothing. Nothing comes up in debug either.

**Comment by SupaHam** [comment]
can I can join the server?

**Comment by Prxmium** [comment]
Sure one moment. I'm hosting from home at the moment so I need to quickly get portforwarding sorted.

**Comment by Prxmium** [comment]
Sorry that took so long. IP is xxx.

**Comment by SupaHam** [comment]
I'm actually clueless!

I've tested every single plugin and exact version you're using, including `git-Spigot-5391d73-00359a1` and everything works perfectly. No empty signs on my end.

Edit: I'd say it's the world, but we created a brand new one and it works just fine and normal non-EssentialsX sign text works as expected.

**Comment by Prxmium** [comment]
Extremely weird. Any way of re-opening the issue so it's visible on the main page? And thanks so much for all your help thus far.

**Comment by SupaHam** [comment]
Could you please try update to the latest spigot. A bit of a long shot but I'm all out of ideas.

**Comment by Prxmium** [comment]
Oddly enough, that seems to have done the trick. I had no clue there was a new version out until you mentioned it. I literally started working on this today. Thanks again, SupaHam. :)

**Comment by SupaHam** [comment]
._. that is literally the worst timing ever. Thanks Obama!

**Comment by drtshock** [comment]
All the links

**Comment by DanteMarshal** [comment]
I have this same issue with EssentialsX Version 2.17.1.0 on git-Spigot-cbd1a1b-009d8af (MC 1.14.4) with LuckPerms 5.0.39 as permissions management plugin.
Same as @Prxmium there's nothing logged in debug mode when I create signs but no sign works.
My Config is correct; signs are enabled.

**Comment by mdcfe** [comment]
@Marshal313 Could you create a new issue for your problem and fill out the appropriate template?

**Comment by DanteMarshal** [comment]
Sure. I'll try changing ess and server settings a little then will post a new issue if nothing worked.
Otherwise will report how it was fixed here.

**Update**
I checked my Server's log, and found that GeoIP plugin is throwing an exception when trying to read its Database : [Server.log](https://gist.github.com/Marshal313/12083d2b44aefb635608871aaa9d7a6a)
So I removed the GeoIP and XMPP jars (with their associated config directories) from my plugins folder and it works fine now.
